railslink/railslink
# tweak marginalia config so it ignores ruby's monitor.rb stack frame

Marginalia and Ruby 2.4/2.5 have an issue where the stack trace includes
Ruby's own monitor.rb and this isn't excluded by the default
configuration so it shows all SQL queries as coming from there. This
patch adds it to the exclusion list.

See https://github.com/basecamp/marginalia/issues/70

test plan:
  - fire up rails server and watch the logs
  - load up the homepage
  - notice that the SQL comments appropriately show where they are
    invoked instead of pointing to monitor.rb
